src/main/java/com/maxmind/geoip2/model/IpRiskResponse.java

```diff
@@ -0,0 +1,85 @@
+package com.maxmind.geoip2.model;
+
+import com.fasterxml.jackson.annotation.JacksonInject;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.maxmind.db.MaxMindDbConstructor;
+import com.maxmind.db.MaxMindDbParameter;
+import com.maxmind.db.Network;
+import com.maxmind.geoip2.NetworkDeserializer;
+
+/**
+ * This class provides the GeoIP2 IP Risk model.
+ */
+public class IpRiskResponse extends IpBaseResponse {
+
+    private final float ipRisk;
```

Comment:
I think the type here will need to be updated as well.

Reply:
Sorry, not sure what exactly needs to change here?

Reply:
Sorry, I was referring to the comment in `MaxMind-DB`. There I mentioned that `ip_risk` should be a `double`. After you update that, this should be changed to a `double` as well (and you should pull in the updated submodule).